packages/orbit-components/src/Icon/Icon.stories.tsx

```diff
@@ -1,129 +1,123 @@
 import * as React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
 
 import IconList from "./IconList";
 import * as Icons from "../icons";
 import { ICON_SIZES, ICON_COLORS } from "./consts";
 import RenderInRtl from "../utils/rtl/RenderInRtl";
 
-export default {
-  title: "Icon",
-};
+import type OrbitIcon from ".";
 
-export const Default = ({ size, color, source, dataTest, ariaLabel, ariaHidden }) => {
-  const Icon = Icons[source];
-  return (
-    <Icon
-      size={size}
-      color={color}
-      dataTest={dataTest}
-      ariaLabel={ariaLabel}
-      ariaHidden={ariaHidden}
-    />
-  );
-};
+const getIcon = source => Icons[source];
+
+type IconPropsAndCustomArgs = React.ComponentProps<typeof OrbitIcon> & { source: string };
+
+const meta: Meta<IconPropsAndCustomArgs> = {
+  title: "Icon",
 
-Default.story = {
-  name: "Default",
   parameters: {
     info: "We use icons to draw attention to specific actions in our products. Visit Orbit.Kiwi for more detailed guidelines.",
   },
-};
 
-Default.args = {
-  size: ICON_SIZES.MEDIUM,
-  color: ICON_COLORS.PRIMARY,
-  source: "Airplane",
-  dataTest: "test",
-  ariaLabel: "label",
-  ariaHidden: true,
-};
+  args: {
+    size: ICON_SIZES.MEDIUM,
+    color: ICON_COLORS.PRIMARY,
+    source: "Airplane",
+    ariaLabel: "label",
+    ariaHidden: true,
+    customColor: "#ABCDEF",
```

Comment:
What's this value for? I believe you're overriding it below with `customColor: "#ff0000"`.

Reply:
good catch, thx. I will keep the "default" customColor. 👍